critiquebrainz/frontend/templates/review/entity/release_group.html

```diff
@@ -20,7 +20,10 @@ <h2 id="title">
       {% set artist = [] %}
       {% for credit in release_group['artist-credit'] %}
         {% if credit.artist %}
-          {% do artist.append('<a href="%s">'|safe % url_for('artist.entity', mbid=credit.artist.id) + credit.artist.name + '</a>'|safe) %}
+          {% do artist.append('<a href="%s">'|safe % url_for('artist.entity', mbid=credit.artist.id) ~ credit.artist.name ~ '</a>'|safe) %}
```

Comment:
As far as I can tell ~ is preferred to + (and this makes it consistent with the RG page header)